tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<UserControl
+    x:Class="DevHome.Customization.Views.AddRepositoriesView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:DevHome.Customization.Views"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+    xmlns:ctControls="using:CommunityToolkit.WinUI.Controls" 
+    xmlns:viewmodels="using:DevHome.Customization.ViewModels"
+    mc:Ignorable="d">
+    <UserControl.DataContext>
+        <viewmodels:FileExplorerViewModel />
+    </UserControl.DataContext>
+    <StackPanel>
+        <TextBlock x:Uid="AddRepositoriesTextBlock"
+               Style="{StaticResource SettingsSectionHeaderTextBlockStyle}"/>
+        <ctControls:SettingsCard
+               x:Uid="AddRepositoriesCard"
+               Grid.Column="1">
+            <Button x:Uid="AddFolderButton" 
+                Style="{ThemeResource AccentButtonStyle}" 
+                Click="{x:Bind ViewModel.AddFolderButton_ClickAsync, Mode=OneWay}" />
+        </ctControls:SettingsCard>
+        <ItemsRepeater Name="ItemsRepeaterForAllRepoPaths" Visibility="Collapsed" ItemsSource="{x:Bind ViewModel.TrackedRepositories, Mode=OneWay}">
+            <ItemsRepeater.ItemTemplate>
+                <DataTemplate>
+                    <ctControls:SettingsCard 
+                        Name="RepoPathCard"
+                        Header="{Binding RepositoryRootPath}"
+                        Margin="{ThemeResource SettingsCardMargin}">
+                        <ctControls:SettingsCard.HeaderIcon>
+                            <FontIcon FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xf12b;"/>
+                        </ctControls:SettingsCard.HeaderIcon>
+                        <StackPanel Name="SettingCardMargin" Orientation="Horizontal">
+                            <DropDownButton Content="{Binding SourceControlProviderDisplayName}">
+                                <DropDownButton.Flyout>
+                                    <MenuFlyout Placement="Bottom">
+                                        <!-- WinUI does not currently allow populating MenuFlyoutItems dynamically within a DataTemplate. -->
+                                        <!-- The items must be statically constructed. -->
+                                        <MenuFlyoutItem Text="Git" Click="AssignSourceControlProviderButton_Click"/>
```

Comment:
This Click can probably become a Command in the ViewModel. You're calling into the VM anyway.

Reply:
Resolved offline. Thanks!